LoopUI/Charts/GlucoseChart.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  GlucoseChart.swift
+//  LoopUI
+//
+//  Copyright © 2019 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import LoopKit
+import SwiftCharts
+
+
+public class GlucoseChart {
+    public init() {
+    }
+
+    public var glucoseUnit: HKUnit = .milligramsPerDeciliter {
+        didSet {
+            if glucoseUnit != oldValue {
+                // Regenerate the glucose display points
+                let oldRange = glucoseDisplayRange
+                glucoseDisplayRange = oldRange
+            }
+        }
+    }
+
+    public var glucoseDisplayRange: (min: HKQuantity, max: HKQuantity)? {
```

Comment:
By using ClosedRange for the return type here you can enforce the relative relationship between the bounds (i.e. guarantee that `max >= min`)